### PR TITLE
Move TF installation to notebooks

### PR DESCRIPTION
**Pull request by numerology**
This PR is part of issue #1750 .
Move TF installation to notebook files to reduce unnecessary environment setup time.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1793)

### CODE REVIEW

**Comment by numerology** [comment]
This one should be okay. @gaoning777 @Ark-kun

File: samples/core/lightweight_component/Lightweight Python components - basics.ipynb [hunk 1]
```diff
@@ -38,8 +38,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Install the SDK\n",
-    "!pip3 install kfp --upgrade\n"
+    "# Install the SDK and tensorflow\n",
+    "!pip3 install kfp --upgrade\n",
+    "!pip3 install tensorflow==1.8.0"
```
Comment (Ark-kun) [inline samples/core/lightweight_component/Lightweight Python components - basics.ipynb]: Is tensorflow used in this notebook?

Comment (numerology) [reply samples/core/lightweight_component/Lightweight Python components - basics.ipynb]: I think yes. In the notebook it uses 
```python
from tensorflow.python.lib.io import file_io
```

**Pull request merged**